### ISSUE TITLE
Add code coverage % calculations to CodeCoverage.java

### ISSUE THREAD

**Issue by adbjo** [johanmoritz/freecol#36]
771a0e33919b73cef83b3ffa23cb5ac2d9587b70 adds branch counts for the currently covered functions and logs them.
It says "77 branch(es) out of 71 were tested" for createLandRegions, doesn't make sense!
Miscalculation by me or potential bug.

TODO, code coverage % calculations.

**Comment by johanmoritz** [comment]
As far as I can see, the #branches is wrong for many of the functions. When I search in VSCode there are _86_ (not 71) branches in "createLandRegions" for example. After going through al of them I get

```java
{
          put("initializeCaches", 23);
          put("writeChildren", 38);
          put("readChild", 28);
          put("setLocation", 27);
          put("getDefaultDisplay", 22);
          put("updateTileImprovementPlans", 25);
          put("readOption", 20);
          put("actionIsValid", 16);
          put("createLandRegions", 86);
          put("findMapPath", 14);
          put("getCost", 26);
      }
```

I can commit this, but would like a second pair of eyes on it first

**Comment by adbjo** [comment]
This is correct, I double checked with Sublime and now get the same numbers as you, thanks for checking.
Used IntelliJ yesterday, must have misread the numbers.

**Comment by adbjo** [comment]
Branch count for BaseCostDecider::getCost changed 26 -> 27 in #46

**Comment by adbjo** [comment]
Solved 6322f23fd1049aa53a5b90bd6d9ab098da387e55
